titan/params/classes.yml

```diff
@@ -133,3 +133,49 @@ classes:
     default:
       world:
         ppl: 1.0
+  exit:
+    type: definition
+    description: Model exit strategies
+    fields:
+      exit_type:
+        type: enum
+        values:
+          - age_out
+          - death
+          - drop_out
+          - none
+      ignore_incar:
+        type: bool
+      age:
+        type: int
```

Comment:
can we add a default here?

Reply:
and for incar

Reply:
I literally don't remember if paraml supports that, but it feels like it would be useful?

Reply:
no defaults as of now :(

Reply:
open an issue on paraml to add default support for def fields? maybe descriptions too?

Reply:
Apparently paraml does support that!